src/Commands/MakeEntityCommand.php

```diff
@@ -320,4 +260,19 @@ protected function validateOnlyApiOption()
             }
         }
     }
+
+    protected function listenEvents(): void
+    {
+        $eventDispatcher = app(EventDispatcher::class);
+
+        $eventDispatcher->listen(
+            events: SuccessCreateMessage::class,
+            listener: fn (SuccessCreateMessage $event) => $this->info($event->message),
+        );
+
+        $eventDispatcher->listen(
```

Comment:
please try to use Event facade for it

```suggestion
        Event::listen(
            events: SuccessCreateMessage::class,
            listener: fn (SuccessCreateMessage $event) => $this->info($event->message),
        );

        Event::listen(
```